.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+install:
+  - pip install -r requirements.txt
+  - python setup.py install
+  - npm install -g tap-spec
```

Comment:
I temporarily disabled it because it made tests that failed appear like they passed (tap-spec expects failed tests to exit 1 and only 1, not, as tsht does by default, with a non-zero value equal to the number of failed tests). This was fixed by https://github.com/kba/tsht/commit/352dcd7e5a3831996fe212c91d38d4d6478d1aa5, I'll reenable it.